tests/robustness/report/report.go

```diff
@@ -46,6 +47,12 @@ func testResultsDirectory(t *testing.T) string {
 	if err != nil {
 		t.Fatal(err)
 	}
+	suffix := fmt.Sprintf("%v", time.Now().UnixNano())
+	overwriteResultsDir, ok := os.LookupEnv("OVERWRITE_RESULTS_DIR")
```

Comment:
No need for `OVERWRITE_RESULTS_DIR` env variable, let's change the defaults.

Reply:
done